### PR TITLE
Stargate

### PR DESCRIPTION
**Pull request by 5A11**
This PR adds Stargate support for fetchai ledger.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by panasevychol — APPROVED** [review]

File: packages/fetchai/agents/random_beacon_feed/aea-config.yaml [hunk 1]
```diff
@@ -35,6 +35,6 @@ type: connection
 config:
   ledger_apis:
     fetchai:
-      address: https://rest-beaconworld.fetch.ai:443
```
Comment (jrriehl) [inline packages/fetchai/agents/random_beacon_feed/aea-config.yaml:38]: I'll check on this, but I believe that Stargateworld does not have a random beacon.

**Review by jrriehl — APPROVED** [review]
Looks good

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
The idea was that the switch to stargate finally resolves the test failures. However, I observe quite a few and it is harder to see now where they come from. I suggest @solarw has a look to address them so that we have a green test run before we merge.

**Comment by 5A11** [comment]
> The idea was that the switch to stargate finally resolves the test failures. However, I observe quite a few and it is harder to see now where they come from. I suggest @solarw has a look to address them so that we have a green test run before we merge.

We have addressed all Stargate related failures. There is none in this PR. 
The failures are either:
a) public DHT related: waiting for prod to be synced with latest ACN changes.
b) multi aea manager related: @solarw is looking into those.
c) odd failures here and there: also @solarw is looking into those.

**Pull request merged**